doc/classes/String.xml

```diff
@@ -181,6 +181,25 @@
 			<argument index="1" name="placeholder" type="String" default="&quot;{_}&quot;" />
 			<description>
 				Formats the string by replacing all occurrences of [code]placeholder[/code] with [code]values[/code].
+				[b]Note:[/b] There must be one and only one [code]_[/code] in the [code]placeholder[/code].
+				[codeblocks]
+				[gdscript]
+				# Use the default placeholder, and a two-dimensional array.
+				var str1 = "name will be replaced to {name}, 0 maps {0}, strength is {strength}"
+				# Then the result1 is "name will be replaced to RobotGuy, 0 maps godot, strength is 9000.91".
+				var result1 = str1.format([["name", "RobotGuy"], [0, "godot"], ["strength", 9000.91]])
+
+				# Use a custom placeholder string "$_", and a dictionary.
+				var str2 = "red=$red green=$green blue=$blue alpha=$alpha"
+				# Then the result2 is "red=0.1 green=0.3 blue=0.5 dict={hello:godoter}".
+				var result2 = str2.format({"red":0.1, "green":0.3, "blue":0.5, "dict":{"hello":"godoter"}}, "$_")
```

Comment:
Almost there, just place back the spaces to the **right** of the ":" characters here.